funnel/models/commentvote.py

```diff
@@ -193,6 +193,7 @@ class Comment(UuidMixin, BaseMixin, db.Model):
                 'user',
                 'title',
                 'message',
+                'parent_id',
```

Comment:
This column needs renaming. It should be `in_reply_to_id`. Also, public-facing id is `in_reply_to_uuid_b58`, a property.

Reply:
It is unfortunate that the property will require loading the comment itself to retrieve its UUID. This is a good case for UUID primary keys.